build.sbt

```diff
@@ -129,7 +129,9 @@ lazy val docs = (project in file("docs"))
     paradoxTheme := Some(builtinParadoxTheme("generic")),
     paradoxProperties in Compile ++= Map(
       "empty" -> "",
-      "version" -> version.value
+      "version" -> version.value,
+      "project.url" -> "https://developer.lightbend.com/docs/paradox/current/",
+      "canonical.base_url" -> "https://developer.lightbend.com/docs/paradox/current/",
```

Comment:
Since in most cases these are going to be the same, I say lets have it default to `project.url` here: https://github.com/lightbend/paradox/blob/master/plugin/src/main/scala/com/lightbend/paradox/sbt/ParadoxPlugin.scala#L89

Also, a good default value for `project.url` is the value of `homepage` sbt setting key.

That would keep the duplication to the minimum, and will do the right thing out of the box.